tools/run-tests.py

```diff
@@ -503,6 +504,31 @@ def run_intl(engine, arch):
     if fails > 0:
         raise Exception('Intl tests failed')
 
+@runner('escargot-debugger', default=True)
+def run_escargot_debugger(engine, arch):
+    Escargot_Debugger_Test_DIR = join(PROJECT_SOURCE_DIR, 'tools', 'debugger', 'tests')
+    Escargot_Debugger_Client = join(PROJECT_SOURCE_DIR, 'tools', 'debugger', 'debugger.py')
+    Escargot_Debugger_Tester = join(PROJECT_SOURCE_DIR, 'tools', 'debugger', 'debugger_tester.sh')
+    print('Running Escargot-debugger test:')
+    test_cases = glob(join(Escargot_Debugger_Test_DIR, '*.js'))
+    test_commands = glob(join(Escargot_Debugger_Test_DIR, '*.cmd'))
+    expected_outputs = glob(join(Escargot_Debugger_Test_DIR, '*.expected'))
+    test_cases.sort()
+    test_commands.sort()
+    expected_outputs.sort()
```

Comment:
We have three sorted sets, but what ensures, that we get the same tests in all (e.g. if one expected is missing, we get wrong order)? I would only use one (`*.cmd`), and use string replace to figure out the corresponding name (e.g. `.cmd` -> `.js`).